### PR TITLE
UI improvements

### PR DESCRIPTION
**Pull request by Bettelstab**
Addresses some of https://github.com/nimiq/safe/issues/61

### CODE REVIEW

File: v-wallet-selector/v-wallet-selector.css [hunk 1]
```diff
@@ -1,10 +1,16 @@
 v-wallet-selector {
-    position: relative;
     display: inline-block;
+    height: fit-content;
```
Comment (danimoh) [inline v-wallet-selector/v-wallet-selector.css:3]: That's still experimental, is that ok?

File: v-wallet-selector/v-wallet-selector.js [hunk 2]
```diff
@@ -64,7 +62,7 @@ export default class VWalletSelector extends MixinRedux(XElement) {
         this._isMenuActive = false
 
         this.vue = new Vue({
-            el: '#vue-wallet-menu',
+            el: `#vue-wallet-menu-${this.attributes.instance}`,
```
Comment (danimoh) [inline v-wallet-selector/v-wallet-selector.js:65]: Can also be just `this.$('.v-wallet-menu')`, then we don't need this instance stuff

File: v-wallet-selector/v-wallet-selector.css [hunk 3]
```diff
@@ -48,20 +47,31 @@ v-wallet-selector [active-wallet-label-mobile]::after {
...
     color: var(--nimiq-blue);
     text-align: left;
 }
 
+v-wallet-selector #vue-wallet-menu-desktop {
+    top: 4.25rem;
+    right: 0;
+}
+
+v-wallet-selector #vue-wallet-menu-mobile {
```
Comment (danimoh) [inline v-wallet-selector/v-wallet-selector.css:66]: Same

File: v-wallet-selector/v-wallet-selector.css [hunk 4]
```diff
@@ -48,20 +47,31 @@ v-wallet-selector [active-wallet-label-mobile]::after {
...
+v-wallet-selector .v-wallet-menu {
     position: absolute;
-    top: 4.25rem;
-    right: 0;
     z-index: 1000;
     color: var(--nimiq-blue);
     text-align: left;
 }
 
+v-wallet-selector #vue-wallet-menu-desktop {
```
Comment (danimoh) [inline v-wallet-selector/v-wallet-selector.css:61]: That's an ugly hack, defining the style within v-wallet-selector for the instance attribute that get's set from outside. Instead, the style should be overwritten from outside.

File: v-wallet-selector/v-wallet-selector.css [hunk 5]
```diff
@@ -48,20 +47,31 @@ v-wallet-selector [active-wallet-label-mobile]::after {
...
+    top: 4.25rem;
+    right: 0;
+}
+
+v-wallet-selector #vue-wallet-menu-mobile {
+    left: 0;
+}
+
+v-wallet-selector #vue-wallet-menu-mobile .wallet-menu {
```
Comment (danimoh) [inline v-wallet-selector/v-wallet-selector.css:70]: same

File: x-request-link/x-create-request-link-modal.js [hunk 6]
```diff
@@ -25,10 +25,9 @@ export default class XCreateRequestLinkModal extends MixinModal(XElement) {
                         <li>
                             <div>Or create a transaction request link:</div>
                             <div class="spacing-top"><x-amount-input></x-amount-input></div>
-                        </li>
-                        <li>
                             <div class="spacing-top">Copy your link:</div>
                             <div class="x-request-link"></div>
+                        </div>
```
Comment (danimoh) [inline x-request-link/x-create-request-link-modal.js:30]: Wrong closing tag?

File: x-transactions/x-transaction.js [hunk 7]
```diff
@@ -7,10 +7,10 @@ export default class XTransaction extends MixinRedux(XElement) {
     html() {
         return `
             <div class="timestamp" title="">pending...</div>
-            <x-identicon sender></x-identicon>
+            <div class="identicon"><x-identicon sender></x-identicon></div>
```
Comment (danimoh) [inline x-transactions/x-transaction.js:10]: Why not add
```js
styles() {
    return [ ...super.styles(), 'identicon' ];
}
```
to `x-identicon`?
Wouldn't that be the clean approach?
or is it really a parent element we need to apply the style to?

File: v-wallet-selector/v-wallet-selector.css (hunk 1 above)
Comment (Bettelstab) [reply v-wallet-selector/v-wallet-selector.css:3]: oh, I didn't know. Maybe not ok

File: x-transactions/x-transaction.js (hunk 7 above)
Comment (Bettelstab) [reply x-transactions/x-transaction.js:10]: I think it is. Not 100% sure now anymore.

**Pull request merged**